timeserio/externals.py

```diff
@@ -22,6 +22,13 @@ def __init__(self, name, *args, **kwargs):
         super().__init__(name, *args, **kwargs)
 
     def __getattr__(self, attr):
+        # Doctest attempts to unwrap everything to see if it can doctest it
+        # so we need to raise an AttributeError in that case rather than the
+        # specific ModuleNotFoundError
+        if attr == "__wrapped__":
```

Comment:
not installing tensorpandas in dev deps lead to discovering this bug with NotFoundModules & doctest. we dont use tensorpandas anywhere else in tests anyway, and it restricts requirements a lot so it's good to get rid of it in the dev dependencies